webtau-coverage/pom.xml

```diff
@@ -0,0 +1,226 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2020 webtau maintainers
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>webtau-parent</artifactId>
+        <groupId>org.testingisdocumenting.webtau</groupId>
+        <version>1.30-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>webtau-coverage</artifactId>
+
+    <properties>
+        <enforcer.skip>true</enforcer.skip>
```

Comment:
I'll have to investigate a bit why this is needed.  Without it, I get the following enforcer failures:
```
[WARNING] Rule 1: org.apache.maven.plugins.enforcer.BanDuplicateClasses failed with message:
Duplicate classes found:

  Found in:
    javax.servlet:javax.servlet-api:jar:3.1.0:test
    org.apache.tomcat.embed:tomcat-embed-core:jar:8.5.56:test
  Duplicate classes:
    javax/servlet/annotation/HttpConstraint.class
    javax/servlet/http/Cookie.class
    javax/servlet/http/HttpSessionBindingEvent.class
    javax/servlet/http/NoBodyResponse.class
    javax/servlet/DispatcherType.class
    javax/servlet/ServletOutputStream.class
    javax/servlet/SessionTrackingMode.class
    javax/servlet/GenericServlet.class
    javax/servlet/http/HttpServletRequestWrapper.class
    javax/servlet/ServletException.class
    javax/servlet/annotation/ServletSecurity$EmptyRoleSemantic.class
    javax/servlet/annotation/ServletSecurity.class
    javax/servlet/ServletRequestAttributeEvent.class
    javax/servlet/ServletRequestWrapper.class
    javax/servlet/ServletRequestEvent.class
    javax/servlet/http/HttpServlet.class
    javax/servlet/ServletContextAttributeEvent.class
    javax/servlet/AsyncEvent.class
    javax/servlet/MultipartConfigElement.class
    javax/servlet/annotation/ServletSecurity$TransportGuarantee.class
    javax/servlet/http/HttpUtils.class
    javax/servlet/ServletSecurityElement.class
    javax/servlet/ServletResponseWrapper.class
    javax/servlet/ServletContextEvent.class
    javax/servlet/http/NoBodyOutputStream.class
    javax/servlet/ServletRegistration$Dynamic.class
    javax/servlet/http/HttpServletResponseWrapper.class
    javax/servlet/http/HttpSessionEvent.class
    javax/servlet/UnavailableException.class
    javax/servlet/annotation/HttpMethodConstraint.class
    javax/servlet/HttpConstraintElement.class
    javax/servlet/ServletInputStream.class
    javax/servlet/HttpMethodConstraintElement.class

  Found in:
    commons-logging:commons-logging:jar:1.2:compile
    org.slf4j:jcl-over-slf4j:jar:1.7.25:test
  Duplicate classes:
    org/apache/commons/logging/Log.class
    org/apache/commons/logging/LogConfigurationException.class
    org/apache/commons/logging/LogFactory.class
    org/apache/commons/logging/impl/SimpleLog$1.class
    org/apache/commons/logging/impl/SimpleLog.class
    org/apache/commons/logging/impl/NoOpLog.class

  Found in:
    org.slf4j:slf4j-simple:jar:1.7.30:compile
    ch.qos.logback:logback-classic:jar:1.1.11:test
  Duplicate classes:
    org/slf4j/impl/StaticMDCBinder.class
    org/slf4j/impl/StaticMarkerBinder.class
    org/slf4j/impl/StaticLoggerBinder.class
```

Reply:
Wonder if replacing snapshot with project.version (another comment) would fix it.

Reply:
One way to find out 😀

Reply:
It did not help.  I suspect what is happening here is there are dependencies that are otherwise never pulled in by the a single module.  Notice how all the clashes involved at least one test dependency so in all likelihood the maven plugin module, main packaging module or any shading modules will not be pulling those in together.

I could go and fix all these but I'm not sure it's worth it given this module's purpose.

Reply:
is it disabled for this module only? I don't know if properties are JVM wide.

Reply:
They're not JVM wide.  They apply only to the module in which they're declared and its child modules.  The latter you see often as they often get declared in the parent/root pom.